Makefile

```diff
@@ -340,7 +340,7 @@ $(FIRECRACKER_BIN): $(FIRECRACKER_DIR)/Cargo.toml
 
 .PHONY: firecracker-clean
 firecracker-clean:
-	- $(FIRECRACKER_DIR)/tools/devtool distclean
+	- $(FIRECRACKER_DIR)/tools/devtool clean
```

Comment:
Surprisingly I don't see any method named `cmd_clean` in firecracker `main` branch https://github.com/firecracker-microvm/firecracker/blob/main/tools/devtool

It only has a `cmd_distclean` that it supports right now.  Do you have a PR for Firecracker as well?

Reply:
You are right.

```
_submodules/firecracker/tools/devtool clean

[31m[Firecracker devtool] Unknown command: clean. Please use `_submodules/firecracker/tools/devtool help` for help.(B[m

make[1]: [Makefile:343: firecracker-clean] Error 1 (ignored)
```